src/main/java/org/jvnet/hudson/test/junit/jupiter/JenkinsRuleExtension.java

```diff
@@ -0,0 +1,36 @@
+package org.jvnet.hudson.test.junit.jupiter;
+
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.api.extension.ParameterContext;
+import org.junit.runner.Description;
+import org.jvnet.hudson.test.JenkinsRecipe;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import javax.annotation.Nonnull;
```

Comment:
Could we use the spotbugs annotation `NonNull` instead of the JSR-305 annotation?

JSR-305 annotations were removed from Jenkins core 2.231 by jenkinsci/jenkins#4604.  Better to use the spotbugs annotation.

```suggestion
import edu.umd.cs.findbugs.annotations.NonNull;
```